src/css/custom.css

```diff
@@ -8,6 +8,22 @@
   --ifm-font-family-base: "Space Grotesk", sans-serif;
   --header-height: 6.5rem;
   --ifm-navbar-height: 6.5rem;
+  --ifm-color-yellow: #fdea2e;
+  --ifm-color-black: #000;
+  --ifm-color-dark: #323335;
+  --ifm-color-dark-800: #121315;
+  --ifm-color-dark-900: #2c2c2c;
+  --ifm-color-white: #ffffff;
+  --ifm-color-gray-300: #f3f4f7;
+  --ifm-color-gray-400: #e4e4e4;
+  --ifm-color-gray-500: #e5e5e5;
+  --ifm-color-gray-600: #e7e7e7;
+  --ifm-color-gray-700: #cececf;
+  --ifm-color-gray-800: #b6b6b7;
+  --ifm-color-linkedin: #0a66c2;
+  --ifm-color-discord: #5865f2;
+  --ifm-color-white-opacity-15: rgba(255, 255, 255, 0.15);
+  --ifm-color-black-opacity-05: rgba(0, 0, 0, 0.05);
```

Comment:
Please give better names like primary, secondary, base etc.

Reply:
https://css-tricks.com/what-do-you-name-color-variables/

Reply:
Hey Amit most of the important colors(primary, secondary) are in tailwind config which are named correctly. These gray colors are used only in few places and they are gray colors so I have kept the names like this.